src/test/java/com/futurecollars/lesson8/zad4/LowerCaseTextFormatterTest.java

```diff
@@ -0,0 +1,17 @@
+package com.futurecollars.lesson8.zad4;
+
+import com.futurcollars.lesson8.zad4.LowerCaseTextFormatter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class LowerCaseTextFormatterTest {
+    @Test
+    void shouldGetLowerCaseText() {
+        // given
+        String text = "uczenie java sprawia mi przyjemność";
```

Comment:
Tutaj to samo, na wejściu powinien być tekst z wielkich liter, a w asercji z małej, po przepuszczeniu przez Twój formatter :)